qiskit/providers/__init__.py

```diff
@@ -462,7 +462,7 @@ def get_translation_stage_plugin(self):
 handles submitting the circuits to the backend to be executed and returning a
 :class:`~qiskit.providers.Job` object. Depending on the type of backend this
 typically involves serializing the circuit object into the API format used by a
-backend. For example, on IBMQ backends from the ``qiskit-ibmq-provider``
+backend. For example, on IBM backends from the ``qiskit-ibm-provider``
 package this involves converting from a quantum circuit and options into a
 `qobj <https://arxiv.org/abs/1809.03452>`__ JSON payload and submitting
 that to the IBM Quantum API. Since every backend interface is different (and
```

Comment:
I'm amazed this is still accurate.

Reply:
Oh actually it isn't I misread it, the qobj part isn't valid anymore the qiskit-ibm-provider is only submitting circuits via qobj over the wire. The rest of that paper about configuration, properties, and pulse defaults are valid though and that's how the backends are described to the provider. I'll update this.

Reply:
Fixed in: https://github.com/Qiskit/qiskit/pull/11513/commits/f4e26f9afa9bd8bc195a4914017809241110d871